x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichPlugin.java

```diff
@@ -60,7 +60,9 @@
 public class EnrichPlugin extends Plugin implements ActionPlugin, IngestPlugin {
 
     static final Setting<Integer> ENRICH_FETCH_SIZE_SETTING =
-        Setting.intSetting("index.xpack.enrich.fetch_size", 10000, 1, 1000000, Setting.Property.NodeScope);
+        Setting.intSetting("xpack.enrich.fetch_size", 10000, 1, 1000000, Setting.Property.NodeScope);
+    static final Setting<Integer> ENRICH_POLICY_MAX_CONCURRENT_RUNS =
+        Setting.intSetting("xpack.enrich.policy.max_concurrent_runs", 50, 1, Setting.Property.NodeScope);
```

Comment:
maybe something like `enrich.max_concurrent_executions`? im not sure we need to add the word policy here.